Doc/whatsnew/3.14.rst

```diff
@@ -145,6 +145,12 @@ pathlib
   :meth:`~pathlib.PurePath.is_relative_to`. In previous versions, any such
   arguments are joined onto *other*.
 
+pty
+___
+
+* Remove deprecated ``pty.master_open`` and ``pty.slave_open``.
+  They had previously raised a :exc:`DeprecationWarning` since Python 3.12.
```

Comment:
For more consistent formatting, suggest replacement, and it's useful to add a "Contributed by" linking back to the issue in case people want to check for more background:

```suggestion
* Remove deprecated :func:`!pty.master_open` and :func:`!pty.slave_open`.
  They had previously raised a :exc:`DeprecationWarning` since Python 3.12.
  Use :func:`pty.openpty` instead.
  (Contributed by Nikita Sobolev in :gh:`118824`.)
```